packages/react/migrations.json

```diff
@@ -104,6 +104,11 @@
       "version": "13.0.0-beta.0",
       "description": "Migrate Storybook to use webpack 5",
       "factory": "./src/migrations/update-13-0-0/migrate-storybook-to-webpack-5"
+    },
+    "update-external-emotion-jsx-runtime-13.4.0": {
+      "version": "13.4.0-beta.0",
```

Comment:
@jaysoo Should I change the targeted version to `13.5.0-beta.0`?

Reply:
[Nx v13.5](https://github.com/nrwl/nx/releases/tag/13.5.0) has just released. So `13.6.0-beta.0` maybe?